src/components/expansion-panel/index.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { memo, PropsWithChildren, useState, useEffect } from 'react';
 
+import { ExpandMore, ExpandLess } from '@material-ui/icons';
```

Comment:
There is a better way to import icons. See https://material-ui.com/components/icons/#usage